packages/httpsnippet-client-api/test/__datasets__/auth-query/har.json

```diff
@@ -7,10 +7,6 @@
           "httpVersion": "HTTP/1.1",
           "method": "GET",
           "queryString": [
-            {
```

Comment:
Removed this because the OpenAPI definition that this `auth-query` test uses doesn't have a `status` query parameter defined.